guide/designing-products/lightning-services.md

```diff
@@ -0,0 +1,128 @@
+
+Below are some popular Lightning services used today. 
+
+## Pay-to-open  
+
+Pay-to-open is when a user pays to have a channel with inbound (receiving) liquidity opened when they receive a payment. A channel is only opened if the receiver has no current channels with inbound liquidity, say when funding a new wallet, or their current channels do not have enough inbound liquidity to receive the payment. This is also referred to as ‘on-demand’ channel opening. 
+
+#### When to use
+
+* When users of your application are not experienced enough to manage their own payment channels.
+* If your application runs mobile lightning nodes that do not operate a dedicated 24/7 Lightning node.
+* If you want your users to fund a wallet application for the first time directly with a Lightning payment. 
+
+#### Trade-offs
+
+* Receiver needs to connect to the service provider's LSP server which has privacy implications.
+* Your application has to lock up funds that may or may not get used. 
+
+#### Applications that use this
+
+* [Phoenix](https://phoenix.acinq.co/)
+* [Breez](https://breez.technology/)
+* [Dunder](https://github.com/hsjoberg/dunder-lsp) 
+* [Blixt](https://blixtwallet.github.io/) (Dunder enabled)
+
+## Swaps
+
+Swaps are when funds are moved between the Bitcoin and Lightning network. They enable the refilling of outbound Lightning channels via on-chain funds. In reverse, they allow users to obtain inbound liquidity with on-chain funds.
+
+Swaps should be non-custodial and trust-minimized using [submarine swaps](https://wiki.ion.radar.tech/tech/research/submarine-swap). Swaps are generally cheaper than manually managing channels. a Trusted swap temporarily has  custody of users' funds, which has many trust and privacy liabilities, and are not recommended. 
+
+#### When to use
+
+* Beginner friendly wallet applications that may not have funds already in Lightning and need to fund via on-chain.
+* If your application allows channel management. 
+
+#### Trade-offs
+
+* When swapping out (Lightning to on-chain) you have to trust the swap server with your prepayment. 
+* Requires two on-chain transactions making them more expensive.
+* Not interoperable as wallets need to understand the swap protocol used. 
+ 
+
+#### Applications that use this
+
+* [Phoenix](https://phoenix.acinq.co/)
+* [Breez](https://breez.technology/)
+* [Lightning Loop](https://lightning.engineering/loop/)
+* [Boltz](https://boltz.exchange/)
+
+## Spend-unconfirmed
+
+Spend-unconfirmed is when users are able to send and receive funds that are not confirmed on the Bitcoin blockchain. This service makes it so users do not have to wait to begin using funds in their application. In times where the mempool is full, this could prevent users waiting long periods of time to access their funds and/or avoid paying high fees to get the transaction confirmed faster.
```

Comment:
```suggestion
Spend-unconfirmed allows user to send and receive funds that are not yet confirmed on the Bitcoin blockchain. This service reduces the wait time for users before they can begin using funds in their application. In times where the mempool is full, this can prevent users from having to wait long periods of time to access their funds, and/or avoid paying high fees to get transactions confirmed faster.
```
A bit hard to parse, tried to slightly simplify.